lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -1356,8 +1362,9 @@ void HdVP2Mesh::_UpdateDrawItem(
                          HdChangeTracker::DirtyRenderTag |
                          HdChangeTracker::DirtyPoints |
                          HdChangeTracker::DirtyExtent |
-                         DirtySelectionHighlight)) {
-        bool enable = drawItem->GetVisible() && !_meshSharedData._points.empty();
+                         DirtySelectionHighlight) ||
+        instancerWithNoInstances) {
```

Comment:
In the instancer section it ignores dirty flags, so unfortunately we have to re-hide the instancer with no instances every frame, but changing from hidden to hidden should have no cost.

Reply:
But if there is nothing else to trigger the sync, we won't run into this code at all. So I would not worry about the cost too much.

Reply:
Maybe protect the bitwise AND with parentheses.